js/ui/text_box/ui.text_editor.base.js

```diff
@@ -719,12 +726,16 @@ const TextEditorBase = Editor.inherit({
                 if(fullName === name) {
                     checkButtonsOptionType(value);
                 }
-                this._$beforeButtonsContainer && this._$beforeButtonsContainer.remove();
-                this._$afterButtonsContainer && this._$afterButtonsContainer.remove();
-                this._buttonCollection.clean();
-                this._renderButtonContainers();
+                this._reRenderButtonContainers();
```

Comment:
```suggestion
                this._cleanButtonContainers();
                this._renderButtonContainers();
```
It looks more clear for me